.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,22 @@
 
 <!-- Describe your changes below in a reasonable amount of detail -->
 
+### Release
+
+Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 
+
+We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines):
+
+* **feat**: A new feature
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing or correcting existing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
+  generation
```

Comment:
Does it make sense to add an example or two here so it is clear what the commit message is supposed to look like?

Reply:
@geoffturk here all documentation complete about commit messages.

https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits


by default, semantic-release uses the rules of `Angularjs`

Reply:
@geoffturk I think so! @wzalazar we've been using simply `feat: some description`. Is this correct? I gather the Angular way has a `scope` which is used like `feat(scope): some description`, but it looks like scope doesn't make sense for us, right?

Reply:
I think the `scope` is used for monorepos. I'm not sure.

Reply:
Alright. Let's ignore it for now.